### ISSUE TITLE
Remote Working Tools

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/remote-working-experiment#17]
https://youtu.be/m6yCKoSetW8

**Comment by nelsonic** [comment]
Atom Teletype: https://blog.atom.io/2017/11/15/code-together-in-real-time-with-teletype-for-atom.html

**Comment by SimonLab** [comment]
Just tried teletype with @Cleop while pairing. It's really much more easier to explain some code than just trying to shout some code on hangout! It's also nice when some tedious code needs to be written on a file as this can be split.
The only annoying point is that you need to ask the one who share the code to open other files for you when you want to change something as I wasn't able to see the list of files on the project and couldn't open them. Other than that teletype with hangout is working for us I think at the momen.
@Cleop any other comments?

**Comment by Cleop** [comment]
Yeah I agree that teletype was really good. I found it more engaging and more productive than just screen-sharing. As @SimonLab said you can work on some boring bits simultaneously and it's faster for gesturing which bits of code you're talking about etc. 

It would be cool if you could navigate between files but I guess that that could mean you lose track of one another or the changes being made.

**Comment by nelsonic** [comment]
```
apm install teletype
```
